src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -806,10 +806,21 @@ namespace ReportSizingManager {
 					} else if ( ZoneEqUnitHeater || ZoneEqVentedSlab ) {  // for unit ventilator the cp value is calculated at 5.05(InitConvTemp) for the child and 60.0C for the unit ventilator //|| ZoneEqUnitVent
 						AutosizeDes = ZoneEqSizing( CurZoneEqNum ).MaxHWVolFlow;
 					} else {
-						CoilInTemp = FinalZoneSizing( CurZoneEqNum ).DesHeatCoilInTemp;
+						if ( ZoneEqSizing( CurZoneEqNum ).SystemAirFlow ) {
+							DesMassFlow = ZoneEqSizing( CurZoneEqNum ).AirVolFlow * StdRhoAir;
+						} else if ( ZoneEqSizing( CurZoneEqNum ).HeatingAirFlow ) {
+							DesMassFlow = ZoneEqSizing( CurZoneEqNum ).HeatingAirVolFlow * StdRhoAir;
+						} else {
+							DesMassFlow = FinalZoneSizing( CurZoneEqNum ).DesHeatMassFlow;
+						}
+						if ( DesMassFlow > 0.0 ) {
+							OutAirFrac = min( StdRhoAir * ZoneEqSizing( CurZoneEqNum ).OAVolFlow / DesMassFlow, 1.0 );
+						} else {
+							OutAirFrac = 0.0;
+						}
+						CoilInTemp = ( 1.0 - OutAirFrac ) * FinalZoneSizing( CurZoneEqNum ).ZoneTempAtHeatPeak + OutAirFrac * FinalZoneSizing( CurZoneEqNum ).OutTempAtHeatPeak;
```

Comment:
What is changed throughout water coil sizing is how the inlet air temperature is calculated. The FinalZoneSizing variable for DesHeatCoilInTemp is wrong at times where the zone air flow rate is <= the OA flow rate. When this happens, the coil inlet temp is = the OA temp. But it's not really, when the water coil is in a parent that operates at a higher air flow rate. So to correct this, the operating coil inlet temp is calculated based on the OA fraction. This is done for water coil inlet T, w, and for the capacity and UA calculation. This method took many iterations as can be seen in the discussion. The result with this method is still a larger coil, but not 4 or 5 times larger as with the initial attempts.